x-pack/plugins/uptime/public/state/actions/ping.ts

```diff
@@ -12,6 +12,8 @@ import {
   GetPingsParams,
 } from '../../../common/runtime_types';
 
+export const clearPings = createAction('CLEAR PINGS');
```

Comment:
I see some actions use SNAKE_CASE and others just use spaces. Which should I be using, or does it matter?

Reply:
I think largely we are not super strict on the formatting of these since in practice no one sees them or needs to do anything with them.